spec/services/search_indexer_spec.rb

```diff
@@ -189,19 +189,25 @@ def scrub(html, strip_diacritics: false)
       topic = Fabricate(:topic, category: category, title: 'this is a test topic')
 
       post = Fabricate(:post, topic: topic, raw: <<~RAW)
-      a https://cnn.com?bob=1, http://stuff.com.au?bill=1 b abc.net/xyz=1
+      a https://abc.com?bob=1, http://efg.com.au?bill=1 b hij.net/xyz=1
+      www.klm.net/?IGNORE=1 <a href="http://abc.de.nop.co.uk?IGNORE=1&ingore2=2">test</a>
       RAW
 
       post.rebake!
       post.reload
       topic = post.topic
```

Comment:
While we're at it :P https://github.com/discourse/discourse/pull/10862/files#r501383631

Reply:
done